src/java/htsjdk/samtools/SAMSequenceDictionary.java

```diff
@@ -215,9 +217,128 @@ public SAMSequenceRecord addSequenceAlias(final String originalName,
         mSequenceMap.put(altName, originalSeqRecord);
         return originalSeqRecord;
     }
-
+    
+    /** 
+     * called by `parseInterval`. Parse a coordinate string, remove the comma look at the suffix (bp,kb,mp,gb) to scale,
+     * verify that the result is a positive integer
+     * @author Pierre Lindenbaum 
+     * @param numStr the coordinate
+     * @return the coordinate as integer
+     */
+    private static int parseCoordinate(final String numStr)
```

Comment:
is SamDictionary the right place for this? isn't this a way to define an Interval? seems like Interval is a more natural place for it. What do you think?